thirdparty/fix_zk_for_aarch64.patch

```diff
@@ -0,0 +1,20 @@
+--- src/c/src/mt_adaptor.c	2022-03-24 11:30:19.018883299 +0000
++++ mt_adaptor.c	2022-03-24 11:30:23.558847831 +0000
+@@ -483,6 +483,9 @@
+ int32_t fetch_and_add(volatile int32_t* operand, int incr)
+ {
+ #ifndef WIN32
++#if defined(__GNUC__)
```

Comment:
Should be `#if defined(__aarch64__)`?

Reply:
from https://issues.apache.org/jira/browse/ZOOKEEPER-1374